app/views/complaints/show.html.erb

```diff
@@ -7,17 +7,40 @@
   <div class="grid-row grid-gap">
     <div class="grid-col-4">
       <div class="summary-section">
-        <h2>Summary</h2>
-        <p>
-          <strong><%= @complaint.grantee %>:</strong>
-          <%= @complaint.summary %>
-        </p>
+        <div class="ct-section-divider">
+          <h2>Summary</h2>
+          <p>
+            <strong><%= @complaint.grantee %>:</strong>
+            <%= @complaint.summary %>
+          </p>
+        </div>
+        <% if @tta_reports.present? %>
+          <%= render "tta_report_list", tta_reports: @tta_reports %> 
+        <% end %>
+        <div class="display-none" id="tta-activity-form">
+          <%= render "tta_activity_report_form", complaint: @complaint %>
+        </div>
+        <div class="ct-section-divider">
+          <p>
+            <span class="text-middle">
+              <svg class="usa-icon text-primary" aria-hidden="true" focusable="false" role="img">
+                <use xlink:href="/assets/img/sprite.svg#add_circle"></use>
+              </svg>
+            </span>
+              <%= link_to "Add TTA Activity Report", "#tta-report-id", {
```

Comment:
[question] Is using `button_to` here to get the semantics of a button way more styling work than having to implement the custom keydown handler to get a link to respond to both Space and Enter?

Reply:
no, i suppose not -- it just feels weird to use button_to and then have to prevent the default of what that functionality wants to do. neither option is great, i suppose, but let do a quick pass of making the button a button

Reply:
i think we need to add some more js to get the icon to be clickable as well ugh